test/Runtime/backtrace.swift

```diff
@@ -1,6 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-build-swift %s -o %t/a.out
-// RUN: not --crash %t/a.out 2>&1 | %{python} %utils/backtrace-check
+// RUN: not --crash %target-run %t/a.out 2>&1 | %{python} %utils/backtrace-check
```

Comment:
We've seen problems elsewhere with `not --crash` + `%target-run` since the *runner* doesn't crash. Check out `test/PlaygroundTransform/Inputs/not.py`.

Reply:
Interesting. Thanks for the pointer. I'll rework it.